### PR TITLE
[Improvement] Remove redundant imports

### PR DESCRIPTION
**Pull request by xleoken**
# Proposed changes

Add `RedundantImport` `EmptyStatement` style rule.

RedundantImport: Checks for redundant import statements. An import statement is considered redundant.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yangzhg** [comment]
Pay attention to the UT failed

**Comment by jackwener** [comment]
I think we should do `format` job together in community after community reach consensus.

related issue #8985

**Comment by leo65535** [comment]
Thanks @yangzhg for your advice , but I can build doris locally.
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for fe-common 1.0-SNAPSHOT:
[INFO] 
[INFO] fe-common .......................................... SUCCESS [ 26.373 s]
[INFO] spark-dpp .......................................... SUCCESS [ 14.103 s]
[INFO] fe-core ............................................ SUCCESS [01:06 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:47 min
[INFO] Finished at: 2022-04-18T07:33:42Z
[INFO] ------------------------------------------------------------------------
***************************************
Successfully build Doris
***************************************
```

**Comment by leo65535** [comment]
> I think we should do `format` job together in community after community reach consensus.
> 
> related issue #8985

Hi @jackwener, we can't do `format` job in just only one issuse, because it's will be a big job, we can split it to several sub tasks, like add `RedundantImport` `EmptyStatement` style rule one by one.

**Comment by jackwener** [comment]
> Hi @jackwener, we can't do format job in just only one issuse, because it's will be a big job, we can split it to several sub tasks, like add RedundantImport EmptyStatement style rule one by one.

Yes, I agree this point. But we're better off dealing with this intensively over a period of time.

Because PR about `format` will influence the git history. We should make `format period` short as possible. 

So, I think consensus is important. 

BTW, Thanks for your contribution❤😃, The doris dev email also is discussing this thing.

**Comment by leo65535** [comment]
> The doris dev email also is discussing this thing.

emm, forgot to subscribe the dev email, I will subscribe the dev email soon.

File: fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BulkLoadJob.java [hunk 1]
```diff
@@ -55,10 +55,6 @@
 import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
 
-import org.apache.commons.lang.StringUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
```
Comment (jackwener) [inline fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BulkLoadJob.java:61]: They shouldn't be removed.

Comment (jackwener) [reply fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BulkLoadJob.java:61]: It's my mistake, they are redundant.

**Review by jackwener — APPROVED** [review]
I build this PR locally successful.
LGTM.
cc @EmmyMiao87

**Review by jackwener — DISMISSED** [review]
We can rebase master for fix the UT build

**Comment by morningman** [comment]
Hi @leo65535 , please rebase the master to pass the check.

**Comment by leo65535** [comment]
Thanks @jackwener @morningman, I had rebased the master branch, but it seems the master branch has some cpp format issue.

https://github.com/apache/incubator-doris/runs/6208605558?check_suite_focus=true

**Comment by jackwener** [comment]
@leo65535 Yes, Be format is in the schedule. Now it's wrong.

**Review by jackwener — APPROVED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
Hi @leo65535 , sorry that I can't merge this PR. We are plan to refactor the codestyle of FE step by step.
The discussion thread is here: https://lists.apache.org/thread/yxjplc7v6sb0kzc9jxr1kl1xy2v8ctr3

I will close this PR, and the issue will be handled, started from this: #9328

**Comment by leo65535** [comment]
Got, thanks @morningman

**Pull request closed without merging**